ebitenutil/loadimage_go_1_16.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2014 Hajime Hoshi
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build (darwin || freebsd || js || linux || windows) && !android && !ios && go1.16
+// +build darwin freebsd js linux windows
+// +build !android
+// +build !ios
```

Comment:
Why are the mobiles opted out? I think this should work in every environment.

Reply:
I've added ios and android - but should I instead remove all os tags?

Reply:
this file is removed now.

Reply:
Ah yeah I intended to remove all the tags first. I acknowledged this file was removed.